src/api/WordsAPI.ts

```diff
@@ -34,3 +36,65 @@ export const useSearch = (word: string | undefined) => {
     enabled: !!word,
   });
 };
+
+const getDirectSearchResult = async (word: string | undefined) => {
```

Comment:
I created a separate suite of helpers / hooks as the `useSearch` hook is used in multiple places (and I only wanted the main search bar to be affected).